docs/manual/java/guide/services/AdditionalRouters.md

```diff
@@ -0,0 +1,103 @@
+# Additional Routers
+
+Since Lagom 1.5.0, is it possible to extend a Lagom Service with additional Play Routers. 
+
+This is particular useful when itegrating Lagom with existing Play Routers, for instance a [Play gRPC Router](https://developer.lightbend.com/docs/play-grpc/0.5.0/lagom/serving-grpc.html) , or any other Play router that you have at your disposal.
+
+In Java, you add an additional router when binding your Lagom Service in your Guice module. You should pass the additonal routers to the `bindService` method together with the Service interface and its implementation. You do this by means of a helper method called `additonalRouters`. 
+
+There are two variants for `additionalRouters`, one that receives a `Class<play.api.routing.Router>` and one that receives a `play.api.routing.Router`.
+
+The first one should be used when your Router has some dependencies and needs to get them injected  by Lagom's runtime DI infrastructure (Guice). In this case, you pass the class and Guice will initialize it with the right dependencies.
+
+
+```java
+public class FileUploadModule extends AbstractModule implements ServiceGuiceSupport {
+    @Override
+    protected void configure() {
+        bindService(
+                HelloService.class, HelloServiceImpl.class,
+                additionalRouter(SomeOtherPlayRouter.class)
+        );
+    }
+}
+```
+
+The second variant should be used be used when the Router does not have any other dependencies and therefore can be immediately passed as an instance.
```

Comment:
Typo. Twice `be used` 😉

Reply:
```suggestion
The second variant should be used when the Router does not have any other dependencies and therefore can be immediately passed as an instance.
```

@ihostage don't you have the option to make GitHub `suggestion`'s on a PR review? (hmm, maybe it's limited to maintainers?)

Reply:
@ignasi35 👍Yes, I can. Just I did not know it 😞 Thanks a lot!